sql/src/main/java/org/opensearch/sql/sql/antlr/SQLSyntaxParser.java

```diff
@@ -32,4 +32,9 @@ public ParseTree parse(String query) {
     return parser.root();
   }
 
+  public boolean containsHints(String query) {
```

Comment:
Longer term, we want to return hints... not whether the parser contains hints.  
Maybe, return the whole list now and check the conditional `hintsParser.root().getChildCount() > 1` in the calling function.

Reply:
Thanks. Changed this in 74f88ae574b06eb63778a511830e0c148cda8539